gutenberg-ramp.php

```diff
@@ -167,6 +169,51 @@ function gutenberg_ramp_require_gutenberg() {
 	add_action( 'admin_init', [ $ramp_compatibility, 'maybe_display_notice' ] );
 }
 
+/**
+ * Display notice about plugin being obsolete.
+ */
+function gutenberg_ramp_admin_notice() {
+	/*
+	 * Only display when user can manage
+	 * options, i.e. only to admins.
+	 */
+	if ( ! current_user_can( 'manage_options' ) ) {
+		return;
+	}
+
+	$gutenberg_ramp = Gutenberg_Ramp::get_instance();
+
+	/*
+	 * Do not display if the plugin is
+	 * not loaded.
+	 */
+	if ( false === $gutenberg_ramp->active ) {
+		return;
+	}
+
+	?>
+	<div class="notice notice-success is-dismissible">
+		<p><?php 
+			printf(
+				esc_html__(
+					'Gutenberg Ramp is deprecated and should be replaced by filters. See %shere%s for more information.',
```

Comment:
Can we avoid "here" as the link text, but make it more descriptive about what it links to instead?